src/error/GraphQLError.js

```diff
@@ -101,7 +101,8 @@ export function GraphQLError( // eslint-disable-line no-redeclare
 
   let _positions = positions;
   if (!_positions && nodes) {
-    _positions = nodes.map(node => node.loc && node.loc.start).filter(Boolean);
+    _positions = nodes.map(node => node.loc && node.loc.start)
```

Comment:
You're right, that's much clearer. I made the changes and both the Flow checks and the tests still seem to pass. (I did change `!=` into `!==`, got a linting error).